tier2/{{cookiecutter.project_slug}}/.github/workflows/contributors.yml

```diff
@@ -0,0 +1,55 @@
+on:
+    push:
+        branches:
+            - main
+    workflow_dispatch:
+
+jobs:
+    update-contributors:
+        runs-on: ubuntu-latest
+        name: Update contributors info in MAINTAINERS.md
+        permissions:
+            contents: write
+            pull-requests: write
+        steps:
+            # Update contributor list
+            - name: Contribute List
+              uses: akhilmhdh/contributors-readme-action@v2.3.10
```

Comment:
This seems like a good upstream dependency although we should keep an eye on it's development health if we plan to use it in a lot of places.

Reply:
:+1: